guide/how-it-works/ecash/fedimint.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: guide
+title: Fedimint
+description: Learn the basics of Fedimint.
+nav_order: 3
+parent: Ecash
+grand_parent: How it works
+permalink: /guide/how-it-works/ecash/fedimint/
+redirect_from:
+ - /guide/ecash/fedimint
+image: https://bitcoin.design/assets/images/guide/how-it-works/ecash/fedimint.jpg
+main_classes: -no-top-padding
+---
+
+<!--
+
+Editor's notes
+
+Explains generally how fedimint works
+
+Illustration sources
+
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/fedimint.jpg"
+   retina = "/assets/images/guide/how-it-works/ecash/fedimint@2x.jpg"
+   mobile = "/assets/images/guide/how-it-works/ecash/fedimint-mobile.jpg"
+   mobileRetina = "/assets/images/guide/how-it-works/ecash/fedimint-mobile@2x.jpg"
+   alt-text = "ecash overview illustration"
+   width = 1600
+   height = 700
+   layout = "full-width"
+%}
+
+# Fedimint
+Fedimint is the first bitcoin-backed ecash protocol. Fedimint decentralizes trust across a federation of guardians, ensuring that no single guardian has complete control over user funds. Fedimint allows for various Federation configurations, each catering to different needs. These configurations include **Federated Mints**, **Federated Single Guardian Mints**, and **Solo Mints (Unfederated Mints)**, each with its strengths and trade-offs.
+
+## Role of Guardians
+Guardians are participants who collaboratively manage a federations operations. They secure the federation's funds using multi-sig wallets, validate transactions through consensus, and control ecash issuance and redemption.
+
+## Role of Gateways
+A gateway is a service that facilitates interactions between the federation (which operates largely off-chain) and the Lightning Network. A gateway acts as a bridge, enabling users within the federation to make payments to and receive payments on the Lightning Network.
+
+**How Gateways Work**: The gateway accepts bitcoin payments on the Lightning Network and converts them into the bitcoin-backed ecash tokens used within the federation. It can also convert bitcoin-backed ecash tokens into bitcoin and send them over the Lightning Network. This is crucial because it allows the users within a federation to interact with the outside bitcoin and Lightning Network.
+
+**Vetted Gateways**: A vetted gateway is one that has been approved by the federation's guardians as reliable and trustworthy. This vetting process helps to ensure that users' transactions are handled by gateways that have a track record of reliability, reducing the likelihood of payment failures.
+
+## Design Best Practices
+To learn more about Fedimint specific design best practices you can check out the [Design best practices]({{ '/guide/how-it-works/ecash/design-best-practices' | relative_url }}) section.
+
+
+## Types of Fedimints
+
+### Federated Mints (4+ Guardians)
+
+{% include picture.html
```

Comment:
I don't understand this diagram. There's a lot going on and I can't really tell what each actor does. I don't think the arrows are enough. One solution could be to have numbers in circles on there, along with a legend or text explanation that references them. Or, break it down into multiple diagrams.

Reply:
I agree. I’d like to revisit the illustrations. I think it would be clearer to depict where each mint or federation is its own circle, and these federations are part of a larger circle representing the Fedimint protocol. The protocol circle would then be connected to the entire bitcoin network with dotted lines representing lightning or on-chain. I’m drawing inspiration from these examples.

![image](https://github.com/user-attachments/assets/b0332db3-5cd6-47f7-a0cc-133e1ae02ef9)![image](https://github.com/user-attachments/assets/88db4734-7841-473d-b2fe-2509d246e8d7)